examples/node_example/testem.json

```diff
@@ -1,9 +1,10 @@
 {
-    "launchers": {
-        "Node": {
-            "command": "mocha tests.js"
-        }
-    },
-    "launch_in_dev": [ "Node" ],
-    "launch_in_ci": [ "Node" ]
+  "launchers": {
+    "Node": {
+      "command": "mocha tests.js -R tap",
+      "protocol": "tap"
```

Comment:
The `node_tap_example` should cover that case already, mind removing the proto and the `-R` tap to ensure only the process exit code is verified.